Automattic/wp-calypso
# Make all secondary text themable

#### Changes proposed in this Pull Request

* Replace the secondary text color `$gray-text-min` with the corresponding CSS custom property `--color-text-subtle`

Scope: entire application

#### Testing instructions

* visually: open up the application make sure it looks exactly the same as on master
* code: is each sass color variable replaced with the correct css var equivalent?
